packages/web3-constants/evm/token.json

```diff
@@ -13,7 +13,10 @@
         "Arbitrum_Rinkeby": "0xB47e6A5f8b33b3F17603C83a0535A9dcD7E32681",
         "xDai": "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d",
         "Celo": "0x471EcE3750Da237f93B8E339c536989b8978a438",
-        "Fantom": "0x21be370d5312f44cb42ce377bc9b8a0cef1a4c83"
+        "Fantom": "0x21be370d5312f44cb42ce377bc9b8a0cef1a4c83",
+        "Aurora": "0xC9BdeEd33CD01541e1eeD10f90519d2C06Fe3feB",
+        "Aurora_Testnet": "",
+        "Aurora_Betanet": ""
```

Comment:
I don't see Aurora_Betanet around other files, do we need it here?

Reply:
I just removed the beta network.